src/features/click-to-play.js

```diff
@@ -998,9 +1016,9 @@ const config = {
     //     },
     //     informationalModal: {
     //         icon: blockedYTVideo,
-    //         messageTitle: 'Enable YouTube previews and reduce privacy?',
+    //         messageTitle: 'Enable all YouTube previews?',
     //         messageBody: 'Showing previews will allow Google (which owns YouTube) to see some of your device’s information, but is still more private than playing the video.',
-    //         confirmButtonText: 'Enable Previews',
+    //         confirmButtonText: 'Enable All Previews',
```

Comment:
I notice the capitalisation of all (`all` vs `All`) is different for these. If that's intended then feel free to ignore, just mentioning it in case it's a mistake.

Reply:
Thank you for pointing this out. But it is intentional, as part of the revision of the [copy](https://app.asana.com/0/0/1203533466418466/1203564528435112/f). The buttons have capitalised text, so my understanding is that we're keeping the format.